invenio/ext/sqlalchemy/__init__.py

```diff
@@ -194,6 +194,14 @@ def apply_driver_hacks(self, app, info, options):
                 'charset': 'utf8mb4',
             })
             event.listen(Pool, 'checkin', autocommit_on_checkin)
+        elif info.drivername == 'postgresql':
+            from psycopg2.extensions import register_adapter, AsIs
+            from werkzeug.local import LocalProxy
+
+            def adapt_proxy(proxy):
+                return AsIs("'%s'" % proxy._get_current_object())
```

Comment:
That instead fails because a string without quotes is interpreted as a column name, which is not found:
```python
ProgrammingError: (psycopg2.ProgrammingError) column "superadmin" does not exist
LINE 3: WHERE "accROLE".name = superadmin AND "user".id = 663
                               ^
 [SQL: 'SELECT count("user".id) > %(count_1)s AS anon_1 \nFROM "user" JOIN "user_accROLE" ON "user".id = "user_accROLE".id_user AND "user_accROLE".expiration >= now() JOIN "accROLE" ON "accROLE".id = "user_accROLE"."id_accROLE" \nWHERE "accROLE".name = %(name_1)s AND "user".id = %(id_1)s'] [parameters: {'id_1': 663, 'name_1': 'superadmin', 'count_1': 0}]
```